.github/workflows/e2e-detox-android.yml

```diff
@@ -0,0 +1,109 @@
+# Can be used to run Detox E2E tests on pull requests for the Mattermost mobile app with low bandwidth
+# by using 'E2E iOS tests for PR (LBW 1)' instead.
+name: Detox E2E Android Tests PR
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+
+  build-android-apk:
+    runs-on: macos-14
```

Comment:
Since Android emulator can run on MacOS and Linux dist machines, we should consider using Linux dist for cost?

@mvitale1989 what do you think?

Reply:
I agree that having it run on standard Linux runners would be better, if possible, to zero out the cost of this job

Reply:
@rahimrahman @mvitale1989 
This workflow wont go into master. As I have mentioned in the PR description, this is just to showcase the changes work and Running the Andorid test on Local is now possible.

I'll remove this workflow. I'll will take your inputs about trying out on Linux runner in upcoming sprints.
Please let me know about the other changes to the Xml files and script.

Reply:
@yasserfaraazkhan Gotcha.  Thanks for clarifying that.